Dockerfile

```diff
@@ -1,31 +1,27 @@
-FROM centos:7
-
-RUN yum -y update && \
-    yum -y install java-1.8.0-openjdk
-
-ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.292.b10-1.el7_9.x86_64
-
-ENV JRE_HOME /usr/lib/jvm/jre
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
 
 RUN useradd -r -g root tomcat
```

Comment:
It's minor, but can we use the longer switches on commands?  --system is a lot more understandable than -r

Reply:
Sorry, I missed that the `tomcat` group is `root`. Same as the other PR: this is way to wide permissions. Please put the `tomcat` user in a `tomcat` group.

Reply:
yeah updated with tomcat group